doc/conf.py

```diff
@@ -0,0 +1,202 @@
+# further.  For a list of options available for each theme, see the
+# documentation.
+#
+html_theme_options = {
+    'navbar_title': 'Ephypype',
+    'bootswatch_theme': "flatly",
+    'navbar_sidebarrel': False,
+    'bootstrap_version': "3",
+    'navbar_links': [
+        ("Gallery", "auto_examples/index"),
+        ("API", "api"),
+        ("Github", "https://github.com/neuropycon/ephypype", True),
+    ],
+    'bootswatch_theme': "united"}
+
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+html_static_path = ['_static']
+
+
+# -- Options for HTMLHelp output ------------------------------------------
+
+# Output file base name for HTML help builder.
+htmlhelp_basename = 'ephypypedoc'
+
+
+# -- Options for LaTeX output ---------------------------------------------
+
+latex_elements = {
+    # The paper size ('letterpaper' or 'a4paper').
+    #
+    # 'papersize': 'letterpaper',
+
+    # The font size ('10pt', '11pt' or '12pt').
+    #
+    # 'pointsize': '10pt',
+
+    # Additional stuff for the LaTeX preamble.
+    #
+    # 'preamble': '',
+
+    # Latex figure (float) alignment
+    #
+    # 'figure_align': 'htbp',
+}
+
+# Grouping the document tree into LaTeX files. List of tuples
+# (source start file, target name, title,
+#  author, documentclass [howto, manual, or own class]).
+latex_documents = [
+    (master_doc, 'ephypype.tex', u'ephypype Documentation',
+     u'Annalisa Pascarela', 'manual'),
+]
+
+
+# -- Options for manual page output ---------------------------------------
+
+# One entry per manual page. List of tuples
+# (source start file, name, description, authors, manual section).
+man_pages = [
+    (master_doc, 'ephypype', u'mne_bids Documentation',
+     [author], 1)
+]
+
+
+# -- Options for Texinfo output -------------------------------------------
+
+# Grouping the document tree into Texinfo files. List of tuples
+# (source start file, target name, title, author,
+#  dir menu entry, description, category)
+texinfo_documents = [
+    (master_doc, 'ephypype', u'ephypype Documentation',
+     author, 'ephypype', 'One line description of project.',
+     'Miscellaneous'),
+]
+
+sphinx_gallery_conf = {
+    'examples_dirs': '../examples',
+    'gallery_dirs': 'auto_examples',
+    'filename_pattern': '^((?!sgskip).)*$',
+    'backreferences_dir': 'generated',
+    'within_subsection_order': FileNameSortKey,
+    'reference_url': {
+        'mne': 'http://mne-tools.github.io/stable/',
+        'numpy': 'http://docs.scipy.org/doc/numpy-1.9.1',
+        'scipy': 'http://docs.scipy.org/doc/scipy-0.17.0/reference',
+        'nipype': 'https://nipype.readthedocs.io/en/latest/api'
+    }
+}
+
+import sys
+import os.path as op
+
+path = op.join(op.dirname(__file__), '../examples/')
+sys.path.insert(0, path)
```

Comment:
@annapasca we should remove `L201` and `L202` after you reformat the power example ...